src/main/java/br/com/alura/comex/service/processadores/Processador.java

```diff
@@ -0,0 +1,10 @@
+package br.com.alura.comex.service.processadores;
+
+import br.com.alura.comex.model.entities.Pedido;
+
+import java.util.List;
+
+public interface Processador {
+
+    List<Pedido> imprimir() throws Exception;
```

Comment:
Não achei esse nome muito legal. Afinal de contas, o `Processador` não imprime. Ele lê os arquivos, recupera os dados, algo do gênero, não é mesmo?

```suggestion
    List<Pedido> recuperarDados() throws Exception;
```